### ISSUE TITLE
Workflow Settings not saving. Reviewers unable to Approve, and forced to use workflow.php config file. [v1.7.3]

### ISSUE THREAD

**Issue by riusi88** [engram-design/Workflow#150]
After updating to version 1.7.3 we are unable to save Workflow settings such as Reviewer User Groups within the Craft dashboard, and our only work around is using the workflow.php config file. Furthermore, second tier Reviewer User Groups are unable to Approve or Reject Drafts, even when using the workflow.php file to configure the Reviewer User Groups.

Image without using workflow.php config file:
<img width="738" alt="image" src="https://user-images.githubusercontent.com/5905213/175397193-811b3b72-f6e3-4a05-9936-5e23f348d8a0.png">
Image with workflow.php config file:
<img width="742" alt="image" src="https://user-images.githubusercontent.com/5905213/175397098-d3f73e92-a2cc-43a6-af15-220096fb6dc8.png">

We believe these two issues might be related, and it may be the case that since Craft is unable to save the Workflow settings of the Reviewer User groups within the project config/db, it is unable to detect that the second tier Reviewer User Group (Marketing Reviewer) has permission to approve and reject a draft that's been previously approved by the first tier Reviewer User Group (Content Owner). Oddly enough the first tier Reviewer User Group is unable to revoke the approval, nor do there Reviewer notes get saved properly, so perhaps it's an issue with the drafts being tied to the user groups properly?

Reviewer User Group Settings are identical:
<img width="449" alt="image" src="https://user-images.githubusercontent.com/5905213/175392349-40c4b328-7085-4ee1-9398-0f2690d5229a.png">

We decided to downgrade to version 1.7.1, and 1.7.2 and see if it fixed any issues. Downgrading fixed the Workflow settings saving issue, and second tier Reviewer User Groups were able to see the approve and reject buttons. However, submitting a draft for approval triggered two previous bugs associated with those versions: https://github.com/verbb/workflow/issues/144 and https://github.com/verbb/workflow/issues/145. So unfortunately that's not a viable solution. 

We also attempted to downgrade Craft to previous versions of 3.7, and try varying version of Workflow 1.7.1, 1.7.2 and 1.7.3 with no success. 

We imagine the root cause may be a code update in between versions 1.7.2 and 1.7.3, since 1.7.2 has all the capabilities we need, but trigger the bugs listed above. 

**Steps to reproduce**

Bug 1, can't save setting:
1. Update to Workflow v1.7.3
2. Create at least 4 User Groups, 1 Editor, 2 Reviewers, 1 publisher
3. Attempt to save Reviewer User Group settings within the Craft Dashboard

Bug 2, tier 2 Reviewer User Group cant approve
1. Editor creates Draft Notifies tier 1 Reviewer User Group
2. Tier 1 Reviewer User Group approves Draft, and notifies tier 2 Reviewer User Group (The notification actually worked, and email was sent properly to tier 2 Reviewer User Group)
3. Tier 2 Reviewer User Group accesses draft edit page, but only sees the "Awaiting approval"
<img width="366" alt="image" src="https://user-images.githubusercontent.com/5905213/175393752-684c0616-a06b-4fd8-ba28-13c783a0ce1f.png">

**Additional info**
- Plugin version:1.7.3
- Craft version:3.7.44
- Multi-site: No

We have a hunch that this all revolves around the fact that Reviewer User Groups can't be saved properly. Hopefully this points you in the right direction.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-3 as 1.7.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by charliestrafe** [comment]
Any idea when the next release is due out? I'm running Craft Pro 4.1.0.2. thanks

**Comment by engram-design** [comment]
Fixed in [1.7.4](https://github.com/verbb/workflow/releases/tag/1.7.4) and [2.0.0-beta.4](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.4)
